### ISSUE TITLE
nav disappears at desktop half-width screen, replaced with nothing

### ISSUE THREAD

**Issue by ashleygwilliams** [npm/newww#1774]
in chrome Version 47.0.2526.111 (64-bit) at ~55% width on a 13" mac book pro
<img width="761" alt="screen shot 2016-01-20 at 3 12 50 pm" src="https://cloud.githubusercontent.com/assets/1163554/12461765/91c59a48-bf88-11e5-873d-613fdae9fce8.png">

**Comment by cowperthwait** [comment]
Is this a dupe of @isaacs ’ [Link to npm On-Site and npm Private Packages from every view of every page #404](https://github.com/npm/issue-tracker/issues/404 "Link to npm On-Site and npm Private Packages from every view of every page #404")?

**Comment by ashleygwilliams** [comment]
nope. that issue is for adding those links. here, the links exist but disappear because of (what seems like) half-implemented media-queries.

**Comment by isaacs** [comment]
It's a dupe of the first half of that issue, which talks about both the docs site and the mobile view.

**Comment by ashleygwilliams** [comment]
hey @aredridel this happens on non-CMS pages as well. it's baked into the styles. since i'm using a module that pulls the styles from this for npm-camp, fixing this is a blocker to that being live (and/or the same bug)

**Comment by aredridel** [comment]
Oh fun. Improper hiding of nav, it's flexbox based. Shouldn't be too hard to fix without disturbing things.
